### PR TITLE
docs: Linkify README, add Windows CLI install commands

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,7 +377,9 @@ See the release section on GitHub.
               than 1.21/1.27.0 instead
         * Binaries are signed with the GPG key
           [`B9E36A7275FC61B464B67907E06FE8F53CDC6A4C`](https://keyserver.ubuntu.com/pks/lookup?search=B9E36A7275FC61B464B67907E06FE8F53CDC6A4C&fingerprint=on&op=index).
-    * there is also a [Chocolatey package](https://community.chocolatey.org/packages/syncthingtray) maintained by yan12125
+    * or, using Winget, type `winget install Martchus.syncthingtray` in a Command Prompt window.
+    * or, using [Scoop[(https://scoop.sh), type `scoop add bucket extras & scoop install extras/syncthingtray`
```
Comment (Martchus) [inline README.md:381]:
```suggestion
    * or, using [Scoop](https://scoop.sh), type `scoop add bucket extras & scoop install extras/syncthingtray`.
```

File: README.md [hunk 2]
```diff
@@ -377,7 +377,9 @@ See the release section on GitHub.
...
+    * or, via this [Chocolatey package](https://community.chocolatey.org/packages/syncthingtray), type `choco install syncthingtray`. (maintained by yan12125).
```
Comment (Martchus) [inline README.md:382]:
```suggestion
    * or, via this [Chocolatey package](https://community.chocolatey.org/packages/syncthingtray), type `choco install syncthingtray`.
```
(For the sake of consistency and brevity.)

File: README.md [hunk 3]
```diff
@@ -575,7 +577,7 @@ Translations for further locales can be added quite easily:
   "Add a new locale". Check that no translations have been lost (except ones which are no
```
Comment (Martchus) [inline README.md:577]: For consistency it would make sense to cover this occurrence as well.

**Review by Martchus — CHANGES_REQUESTED** [review]
Thank you, I think the links are very useful (and somehow it never occurred to me that one can easily add them using an URL with just a fragment).

I can also add the additional Windows option to the website later (or you can create a PR against the gh-pages branch).

File: README.md (hunk 3 above)
Comment (rasa) [reply README.md:577]: I missed this one. Good catch.

**Review by Martchus — APPROVED** [review]
Thanks, I'll merge it.

**Pull request merged**